lib/generators/boring/brakeman/install/install_generator.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Boring
+  module Brakeman
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds Brakeman to the application'
+      source_root File.expand_path('templates', __dir__)
+
+      RUBY_VERSION_FILE = '.ruby-version'
+
+      DEFAULT_RUBY_VERSION = '.ruby-version'
+
+      class_option :skip_config, type: :boolean, aliases: '-c',
+                                                desc: 'Skip adding Brakeman configuration. Defaults to false'
+
+      class_option :github, type: :boolean, aliases: '-gh',
+                                                desc: 'Add GitHub Actions configuration. Defaults to false'
+
+      class_option :gitlab, type: :boolean, aliases: '-gl',
+                                                desc: 'Add GitLab CI configuration. Defaults to false'
+
+      class_option :ruby_version, type: :string, aliases: '-v',
+                                                 desc: 'Tell us the ruby version which you use for the application. Default to Ruby #{DEFAULT_RUBY_VERSION}, which will cause the action to use the version specified in the #{RUBY_VERSION_FILE} file.'
+
+      def add_brakeman_gem
+        say 'Adding Brakeman gem', :green
+
+        gem 'brakeman', group: :development
+      end
+
+      def configure_brakeman
+        return if options[:skip_config]
+
+        say 'Copying Brakeman configuration', :green
```

Comment:
instead of saying "Copying" "Setting up Breakman" could be more appropriate

Reply:
Brakeman was configured under scan in rails repo, so I thought it was standard.